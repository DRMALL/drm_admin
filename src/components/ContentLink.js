

import React from 'react'
import { Link } from 'react-router'
import Icon from 'antd/lib/icon'

const ContentLink = ({path, tip}) => (
    <Link to={ path } className='content-link' >
      <div className='content-link-block' ><Icon type="plus" /></div>
      <div className='content-link-text' >{ tip }</div>
    </Link>
  )

export default ContentLink
