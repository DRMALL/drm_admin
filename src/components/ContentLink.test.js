

import ContentLink from './ContentLink'
import renderer from 'react-test-renderer'
import React from 'react'

it('ContentLink test', () => {
  const path = '/trouble/new'
  const tip = '创建新信息'
  const tree = renderer.create(<ContentLink path={ path } tip={ tip } />)
  expect(tree).toMatchSnapshot()
})
