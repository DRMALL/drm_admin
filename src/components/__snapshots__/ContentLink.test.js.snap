// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentLink test 1`] = `
<a
  className="content-link"
  onClick={[Function]}
  style={Object {}}
>
  <div
    className="content-link-block"
  >
    <i
      className="anticon anticon-plus"
    />
  </div>
  <div
    className="content-link-text"
  >
    创建新信息
  </div>
</a>
`;
